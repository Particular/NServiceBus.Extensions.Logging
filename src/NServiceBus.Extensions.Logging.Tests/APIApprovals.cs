namespace NServiceBus.Extensions.Logging.Tests
{
    using Logging;
    using NUnit.Framework;
    using Particular.Approvals;
    using PublicApiGenerator;

    [TestFixture]
    public class APIApprovals
    {
        [Test]
        public void Approve_NServiceBusExtensionsLogging()
        {
            var options = new ApiGeneratorOptions
            {
                ExcludeAttributes = new[] { "System.Runtime.Versioning.TargetFrameworkAttribute" }
            };

            var publicApi = ApiGenerator.GeneratePublicApi(typeof(ExtensionsLoggerFactory).Assembly, options);
            Approver.Verify(publicApi);
        }
    }
}