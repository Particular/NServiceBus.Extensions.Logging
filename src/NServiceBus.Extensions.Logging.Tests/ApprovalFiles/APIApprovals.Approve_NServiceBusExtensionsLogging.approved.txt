namespace NServiceBus.Extensions.Logging
{
    public class ExtensionsLoggerFactory : NServiceBus.Logging.ILoggerFactory
    {
        public ExtensionsLoggerFactory(Microsoft.Extensions.Logging.ILoggerFactory loggerFactory) { }
        public NServiceBus.Logging.ILog GetLogger(System.Type type) { }
        public NServiceBus.Logging.ILog GetLogger(string name) { }
    }
}